internal/server/engine.go

```diff
@@ -4,32 +4,40 @@ import (
 	"github.com/dhis2-sre/im-manager/internal/middleware"
 	"github.com/dhis2-sre/im-manager/pkg/health"
 	"github.com/gin-contrib/cors"
+	"github.com/gin-contrib/logger"
 	"github.com/gin-gonic/gin"
 	redocMiddleware "github.com/go-openapi/runtime/middleware"
+	"github.com/rs/zerolog"
```

Comment:
should we adopt https://go.dev/blog/slog in Go 1.21 instead?

Reply:
Yes, we should :smile: